src/xdocs/cmdline.xml.vm

```diff
@@ -73,6 +73,14 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           <code>-o file</code> - specify the file to output
           to.
         </li>
+        <li>
+          <code>-s</code> - print xpath suppressions at the file's line and column position. The option
+          cannot be used other options and requires exactly one file to run on to be specified.
+        </li>
+        <li>
+          <code>-tabWidth</code> - set the length of the tab character. Used only with <code>-s</code> option.
+          Default value is 8.
+        </li>
         <li>
           <code>-t, --tree</code> - print Abstract Syntax Tree(AST) of the checked file. The option
           cannot be used other options and requires exactly one file to run on to be specified.
```

Comment:
@rnveach 
Shouldn't it be "The option cannot be used **with** other options and requires exactly one file to run on to be specified"?

Reply:
@timurt Yes.

Reply:
done